packages/atomic/src/components/common/layout/sections.ts

```diff
@@ -1,4 +1,6 @@
-export const resultSectionTags = [
+import {isElementNode} from '../../../utils/utils';
+
+export const resultSectionTags = new Set([
```

Comment:
I prefer making this a `set` since its purpose is to validate whether a tag name is a section.